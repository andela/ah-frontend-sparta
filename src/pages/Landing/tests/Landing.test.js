import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../../store';
import { Landing } from '../index.js';

describe('Landing page', () => {
	const props = {
		articles: [],
		fetchArticles: jest.fn()
	};
	it('should render without crashing', () => {
		const wrapper = renderer.create(
			<Provider store={store}>
				<Landing {...props} />
			</Provider>
		);
		expect(wrapper.toJSON()).toMatchSnapshot();
	});
});
