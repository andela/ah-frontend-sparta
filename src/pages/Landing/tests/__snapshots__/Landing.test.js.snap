// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing page should render without crashing 1`] = `
<div>
  <div
    className="page-banner-image"
  >
    <img
      alt="Landing image"
      className="LandingImage"
      src={Object {}}
    />
  </div>
  <div
    className="small-screen"
  >
    <div>
      <form>
        <input
          placeholder="SEARCH"
          type="text"
        />
      </form>
    </div>
    <div>
      <a
        href="/login"
      >
        Account
      </a>
    </div>
  </div>
  <div
    className="wrapper"
  >
    <div>
      <div>
        No Posts where Found !!!!
      </div>
    </div>
    <div>
      <h1>
        Trending
      </h1>
      <br />
      <hr />
      <br />
      <div>
        No Trending Articles where Found !!!!
      </div>
    </div>
  </div>
  <br />
  <br />
  <br />
</div>
`;
